os/binfmt/binfmt_exec.c

```diff
@@ -149,7 +150,7 @@ int exec(FAR const char *filename, FAR char *const *argv, FAR const struct symta
 	start_addr = kmm_memalign(size, size);
 
 	if (!start_addr) {
-		berr("ERROR: Failed to allocate RAM partition\n");
+		berr("%s #1\n",clog_message_str[CMN_LOG_ALLOC_FAIL]);
```

Comment:
Let's use current logs instead of #1 and #2.
I mean
```
berr("$s RAM Partition\n", clog_message_str[CMN_LOG_ALLOC_FAIL]);
```

Reply:
ok, I will change all the changes in the similar way.